docker-compose-develop.yml

```diff
@@ -1,15 +1,41 @@
 version: "3"
 services:
-  test:
+  develop:
     build: .
+    ports:
+      - "30518:30518"
     command: develop
     environment:
       MONGO_DATABASE: 'doc-importer-tasks'
-      CT_REGISTER_MODE: auto
+      MONGO_PORT_27017_TCP_ADDR: mymachine
```

Comment:
This is out of scope, and a bad solution to the problem you're trying to solve, as it won't work without custom hosts.

Reply:
Indeed, updated.